src/Broadway/EventStore/EventStoreInterface.php

```diff
@@ -28,6 +28,8 @@ public function load($id);
     /**
      * @param mixed                      $id
      * @param DomainEventStreamInterface $eventStream
+     *
+     * @throws EventStoreException
```

Comment:
This was added because the exception was undocumented, its more of a bugfix.

I can add both, just to clearify?